default.nix

```diff
@@ -780,6 +780,7 @@ rec {
           nixpkgs.nix-update
           nixpkgs.rlwrap # for `rlwrap moc`
           nixpkgs.difftastic
+          nixpkgs.openjdk nixpkgs.z3 nixpkgs.jq # for viper dev
```

Comment:
back this out for now?

Reply:
FYI, these dependencies are needed not only for "viper dev" but also for using viper (e.g., as part of the IDE extension). 

There's probably no ready-to-use Nix package for ViperServer itself. What would it take to create one? (Not urgent)

Reply:
See line 753 :-)